src/main/java/uk/gov/hmcts/reform/professionalapi/domain/entities/Organisation.java

```diff
@@ -73,7 +114,39 @@ public String getName() {
         return paymentAccounts;
     }
 
+    public List<ContactInformation> getContactInformation() {
```

Comment:
adding all these getters is somewhat redundant given that @Getter has also been added to the class, but not an issue